pkg/persistentcache/persistentcache.go

```diff
@@ -20,7 +20,7 @@ import (
 // use the check_id formed with $check_name:$hash
 func getFileForKey(key string) (string, error) {
 	// Invalid characters to clean up
-	invalidChars, err := regexp.Compile("[<>:\"\\|?/\\*]")
```

Comment:
Nit: Compile the pattern once with ``var invalidCharsPattern = regexp.MustCompile(`[^a-zA-Z0-9_-]`)`` at the top of the file

Reply:
Done!